pheps/phep-9999.md

```diff
@@ -0,0 +1,156 @@
+
+This makes the usage of `astropy` ideal as not only are they backed by a larger community than PyHC, they have a core team that maintains and extends the library, and are funded by several NASA missions.
+This reduces the burden on the PyHC members to maintain larger codebases.
+Furthermore, `astropy` provides many key components that would allow further standardization in future.
+
+From a user point of view, reducing the different Application Programming Interfaces (APIs) that have to be learnt within the PyHC ecosystem reduces time spent doing the science that they are interested in.
+It also adds roadblocks to interoperability if the user needs to change from one sub-domain to another.
+The focus within PyHC should be to improve the user experience and reduce the walls that block the ever going need to do multi-instrument science.
+
+There is a community worry around funding, however in this authors opinion, the current landscape provides funding opportunity given a targeted proposal.
+For example, "F.7 - Support for Open-Source Tools, Frameworks, and Libraries" or "F.8 - Supplemental Open-
+Source Software Awards" or "B.20 - Heliophysics Tools and Methods" within their stated scope would allow proposals that can fund this refactor work.
+
+There has also been worry that reducing the number of implementations would reduce the innovation within the PyHC community.
+The author disagrees as the innovation that is driving the heliophysics field is not in adding more APIs on how to create time objects.
+The scientists creating new innovative methods or finding major discoveries do not want to have to deal with switching APIs to handle very similar time objects.
+All that does is slow them down.
+
+One of the most controversial events within the Linux ecosystem (in the authors short lifetime) was the adoption of [systemd](https://systemd.io/).
+Many of the arguments around that time where similar, it reduces innovation, its too complex, users have to learn a new system.
+However, the maintainers of each major linux distribution embraced systemd as it provided a solid foundation to reduce their own maintenance burden but also having one system meant that one could find detailed documentation or common problems and how to solve them via an internet search.
+
+The adoption of `astropy` is in the authors opinion, the same.
+It provides a new foundation that not only reduces the amount of code within PyHC that handles the same functionality, it allows users to focus on the parts of their work that matter.
+Not only would adding more features to `astropy` benefit the astronomy community, it would in turn benefit the PyHC ecosystem.
+
+# Specification
+
+A dedicated implementation is most likely out of scope for a PHEP, as each package will require bespoke changes after an audit.
+
+However, there will be a few common strategies that can be summarized below:
```

Comment:
Adding the strategies for making this effort is really helpful, so thank you for adding it!